frontend/src/app/sample/sample.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute, Router } from "@angular/router";
+import { AethelApiService } from "../shared/services/aethel-api.service";
+import { filter, map } from "rxjs";
+import { AethelDetailResult, LexicalPhrase } from "../shared/types";
+
+@Component({
+    selector: "pp-sample",
+    templateUrl: "./sample.component.html",
+    styleUrl: "./sample.component.scss",
+})
+export class SampleComponent {
+    private sampleName = this.route.snapshot.params["sampleName"];
+    private sample$ = this.apiService.sampleResult$(this.sampleName);
+    public sampleResult$ = this.sample$.pipe(
+        map((response) => response?.result),
+        filter((result): result is AethelDetailResult => result !== null),
+    );
+
+    constructor(
+        private route: ActivatedRoute,
+        private apiService: AethelApiService,
+        private router: Router,
+    ) {}
+
+    public routeToAethel(items: LexicalPhrase["items"]): void {
+        const combined = items.map((item) => item.word).join(" ");
+        this.router.navigate(["/æthel"], {
```

Comment:
using æ in the url makes me uneasy, but I can't fully explain why

Reply:
Me too, and hearing that you find it strange as well, I'll make sure to change it.

(Fun fact: using æ in the template confuses the Angular language service, apparently.)